modules/modules.go

```diff
@@ -380,3 +424,22 @@ func initDependencies() error {
 
 	return nil
 }
+
+// EnterSleepMode enables or disables sleep mode for all the modules.
+func EnterSleepMode(enabled bool) {
```

Comment:
`EnterSleepMode(false)` reads a bit weird.
How about `SetSleepMode(false)`? Or, to more accurately reflect what is happening: `SleepAllModules(false)` or `SetSleepOnAllModules(false)`.

Possibly we could change this on the module too, to match this better: `SetSleep(false)`.